src/Components/Boilerplates/BoilerplatesTable.js

```diff
@@ -3,37 +3,36 @@ import React, { useMemo } from "react";
 import { Link } from "react-router-dom";
 import { useTable } from "react-table";
 import useBuildOrganizationsLink from "../../Hooks/useBuildOrganizationsLink";
-import './BoilerplatesTable.css'
+import "./BoilerplatesTable.css";
```

Comment:
It looks like the majority of the changes in this files are formatting, probably a linter. @Jess-White maybe we should sync on how we can have a configuration setup so that I'm using the same one!